packages/itwin/grouping-mapping-widget/src/widget/components/property-grid/PropertyView.tsx

```diff
@@ -35,7 +34,7 @@ export interface PropertyViewProps extends SharedRendererProps {
  * @public
  */
 export const PropertyView = (props: PropertyViewProps) => {
-  const context = React.useContext(GroupQueryBuilderContext);
+  const context = React.useContext(PropertyGridWrapperContext);
```

Comment:
`React.useContext` -> `useContext`. Even better would be to have a `usePropertyGridWrapper` hook.

Reply:
Created a hook, destructured context object as well